package.json

```diff
@@ -4,9 +4,11 @@
   "private": true,
   "dependencies": {
     "@material-ui/core": "^4.9.14",
+    "@material-ui/icons": "^4.9.1",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "material-ui-icons": "^1.0.0-beta.36",
```

Comment:
This is my bad but we only need to install the `@material-ui/icons` package, not the `material-ui-icons`, so can you delete line 11?
Then run an `npm start` again and make sure the phone icon shows up